packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -55,21 +55,30 @@ async function addStoryWithFont(title) {
   await page.reload();
 }
 
+function isPlatformMacOS() {
+  return page.evaluate(() => {
+    const { platform } = window.navigator;
+    return platform.includes('Mac') || ['iPad', 'iPhone'].includes(platform);
+  });
+}
+
 async function toggleDevTools() {
   const areDevToolsOpen = Boolean(await page.$('#web-stories-editor textarea'));
 
   // Cancel whatever current action to ensure the below shortcut works.
   await page.keyboard.press('Escape');
   await page.click('[aria-label="Web Stories Editor"]');
 
-  await page.keyboard.down('Meta');
+  const isMacOS = await isPlatformMacOS();
+
+  await page.keyboard.down(isMacOS ? 'Meta' : 'Control');
   await page.keyboard.down('Shift');
   await page.keyboard.down('Alt');
   await page.keyboard.down('J');
   await page.keyboard.up('J');
   await page.keyboard.up('Alt');
   await page.keyboard.up('Shift');
-  await page.keyboard.up('Meta');
+  await page.keyboard.down(isMacOS ? 'Meta' : 'Control');
```

Comment:
Took me a looooong time to figure that one out 🤦 

I confused it with the "Mod" key we use for keyboard shortcuts that does this switch automatically. But here we have to do this ourselves apparently.

Reply:
This looks very similar too 
https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/karma-fixture/src/events.js#L556-L582

Any chance we could reuse it.

Reply:
Oh, nice find. And good question.

The only way to reuse this would be by copying it completely to our `e2e-test-utils` package.

Since we rarely use the keyboard in e2e tests and this here is the first time we've needed the Mac checks, I don't currently see the need for doing that. But we can do that once we need it more often.